src/renderers/native-rt/ReactNativeRTEventEmitter.js

```diff
@@ -40,4 +41,7 @@ var ReactNativeRTEventEmitter = {
   },
 };
 
-module.exports = ReactNativeRTEventEmitter;
+BatchedBridge.registerCallableModule(
+  'RTEventEmitter',
+  ReactNativeRTEventEmitter,
+);
```

Comment:
nit: Seems inconsistent that this self-registers when other similar things are registered by a *Injection or *Entry file.

Reply:
I rationalized this with that `require('InitializeJavaScriptCore')` sets itself up. The only other injection for RT is the batching injection.

I think ultimately we'll probably just want to merge this all into a single file and avoid the funny abstractions and indirections.

Even for RN proper we can probably remove a few layers of indirections.